packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -24,7 +24,6 @@ Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(deface) < 2.0
```

Comment:
you droped the `BuildRequires`, but the `Requires` is still there?

Reply:
My mistake, fixed now.